Update Get-CsOnlineUser.md (#9700)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

## PARAMETERS

### -AccountType

> New parameter added for Get-CsOnlineUser which indicates the user type. The possible values for AccountType are: 

Reviewer: ```suggestion
This parameter is added to Get-CsOnlineUser to indicate the user type. The possible values for the AccountType parameter are: 
```

> - 'User' - to query for user accounts
> - 'ResourceAccount' - to query for app endpoints or resource accounts
> - 'Guest' - to query for guest accounts
> - 'Unknown' - to query for user type that is not known

Reviewer: ```suggestion
- `User` - to query for user accounts.
- `ResourceAccount` - to query for app endpoints or resource accounts.
- `Guest` - to query for guest accounts.
- `Unknown` - to query for a user type that is not known.
```
